packages/marketplace/src/components/pages/client-app-detail/client-app-detail.tsx

```diff
@@ -92,11 +117,16 @@ const ClientAppDetail: React.FC<ClientAppDetailProps> = () => {
   const isLoadingAppDetail = useSelector(selectAppDetailLoading)
   const loginType = useSelector(selectLoginType)
   const isAdmin = useSelector(selectIsAdmin)
+  const error = useSelector(selectAppDetailError)
+
   const isInstallBtnHidden = loginType === 'CLIENT' && !isAdmin
   // selector selectAppDetailData return {} if not data
   const unfetched = Object.keys(appDetailData).length === 0
   const { id = '', installedOn = '' } = appDetailData
 
+  React.useEffect(handleApplyAppDetailsFromLocalStorage(dispatch, appid), [dispatch])
```

Comment:
Do we need to add appid to it?
```
React.useEffect(handleApplyAppDetailsFromLocalStorage(dispatch, appid), [dispatch, appId])
```

Reply:
I think No, it should effect once time to get localStorage data.